### DISCUSSION TITLE
Export data as excel

### DISCUSSION THREAD

**Discussion by clhambone** [trimble-oss/dba-dash#775] [General]
I know within the DBADash gui there is a way to export data to excel but is there a way to automate that?  I looked for other general discussions but could not find anything related to it so that is why I started a new discussion.

**Comment by DavidWiseman** [comment]
There isn't a way to do this by the app but you could probably use [Invoke-DbaQuery](https://docs.dbatools.io/Invoke-DbaQuery) with Export-Excel to accomplish this.  The query used to generate the report in the app will have an associated stored procedure in the repository database you can run.  This wouldn't include any formatting the app might do.

Hope this helps

**Comment by clhambone** [comment]
Thanks
